pyproject.toml

```diff
@@ -21,11 +21,12 @@ environment = 'RUSTUP_TOOLCHAIN="stable"'
 
 [tool.cibuildwheel.linux]
 before-all = "yum install -y wget && {package}/tools/install_rust.sh"
-environment = 'PATH="$PATH:$HOME/.cargo/bin" CARGO_NET_GIT_FETCH_WITH_CLI="true" RUSTUP_TOOLCHAIN="stable"'
+environment = 'PATH="$PATH:$HOME/.cargo/bin" CARGO_NET_GIT_FETCH_WITH_CLI="true" RUSTUP_TOOLCHAIN="1.73"'
 repair-wheel-command = "auditwheel repair -w {dest_dir} {wheel} && pipx run abi3audit --strict --report {wheel}"
 
 [tool.cibuildwheel.macos]
 repair-wheel-command = "delocate-wheel --require-archs {delocate_archs} -w {dest_dir} -v {wheel} && pipx run abi3audit --strict --report {wheel}"
+environment = 'RUSTUP_TOOLCHAIN="1.73"'
```

Comment:
This still has the pin for `RUSTUP_TOOLCHAIN` 1.73 pin for Linux builds - did you mean to keep that?

Reply:
Oops, yeah I meant to change this back to stable. My original intent was to pin all the rust versions to 1.73 but after struggling to get win32 builds working I switched to only pinning the macos builds

Reply:
I reverted the Linux change in https://github.com/Qiskit/qiskit/pull/11267/commits/37a9a5219f7efbaa13ce57bdf8b1220c4b36d23f